ThunderCatsTests/UIColorTests.m

```diff
@@ -30,101 +30,70 @@
 #import <XCTest/XCTest.h>
 #import "UIColor+TCAdditions.h"
 
-
 @interface UIColorTests : XCTestCase
 
 @end
 
-
 @implementation UIColorTests
 
 - (void)testColorWithHexString
 {
-    UIColor *color1 = [UIColor tc_colorWithHexString:@"000000"];
-    UIColor *color2 = [UIColor blackColor];
-    
-    CGFloat r1, r2, g1, g2, b1, b2, a1, a2;
-    [color1 getRed:&r1 green:&g1 blue:&b1 alpha:&a1];
-    [color2 getRed:&r2 green:&g2 blue:&b2 alpha:&a2];
+    UIColor *blackColor = [UIColor tc_colorWithHexString:@"000"];
     
-    XCTAssertTrue(r1-r2 == 0 && g1-g2 == 0 && b1-b2 == 0 && a1-a2 == 0);
+    CGFloat red, green, blue, alpha;
     
-    color2 = [UIColor redColor];
-    [color2 getRed:&r2 green:&g2 blue:&b2 alpha:&a2];
+    [blackColor getRed:&red green:&green blue:&blue alpha:&alpha];
     
-    XCTAssertFalse(r1-r2 == 0 && g1-g2 == 0 && b1-b2 == 0 && a1-a2 == 0);
+    XCTAssertEqualWithAccuracy(red, 0.0/255.0, 0.05,  @"Expected red value of 0, but got %f", red);
```

Comment:
Copy & pasted "red" all the way through here, I see.  Should be red, green, blue, and alpha.